pulpcore/cli/rpm/uln_remote.py

```diff
@@ -0,0 +1,56 @@
+import click
+
+from pulpcore.cli.common.context import PulpContext, pass_pulp_context
+from pulpcore.cli.common.generic import (
+    common_remote_create_options,
+    common_remote_update_options,
+    create_command,
+    destroy_command,
+    href_option,
+    label_command,
+    label_select_option,
+    list_command,
+    name_option,
+    show_command,
+    update_command,
+)
+from pulpcore.cli.common.i18n import get_translation
+from pulpcore.cli.rpm.context import PulpUlnRemoteContext
+
+translation = get_translation(__name__)
+_ = translation.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "remote_type",
+    type=click.Choice(["rpm"], case_sensitive=False),
+    default="rpm",
+)
+@pass_pulp_context
+@click.pass_context
+def uln_remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
+    if remote_type == "rpm":
+        ctx.obj = PulpUlnRemoteContext(pulp_ctx)
+    else:
+        raise NotImplementedError()
```

Comment:
I would have expected, that we could add a new type to the `rpm remote` command instead.

Reply:
So you mean `rpm remote uln <options>` ?

Reply:
Almost: `pulp rpm remote --type uln <subcommands> <options>`

Reply:
Fixed.